samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
-);
+ReactDOM.render(
+    <React.StrictMode>
```

Comment:
![image](https://user-images.githubusercontent.com/59052547/184617689-3513bb87-b319-4a9d-a342-cc0a4df22a41.png)

Image name is wrong:
"screen" but not 3 time e

Reply:
Removed additional unnecessary images and updated file names